src/listen/websocket.rs

```diff
@@ -233,66 +339,13 @@ impl<'a> StreamRequestBuilder<'a, Receiver<Result<Bytes>>, DeepgramError> {
     }
 }
 
-fn options_to_query_string(options: &Options) -> String {
-    let serialized_options = SerializableOptions::from(options);
-    serde_urlencoded::to_string(serialized_options).unwrap_or_default()
-}
-
 impl<S, E> StreamRequestBuilder<'_, S, E>
 where
     S: Stream<Item = std::result::Result<Bytes, E>> + Send + Unpin + 'static,
     E: Send + std::fmt::Debug,
 {
     pub async fn start(self) -> Result<Receiver<Result<StreamResponse>>> {
-        // This unwrap is safe because we're parsing a static.
-        let mut url = self.stream_url;
-        {
-            let mut pairs = url.query_pairs_mut();
-
-            // Add standard pre-recorded options
-            if let Some(options) = &self.options {
-                let query_string = options_to_query_string(options);
-                let query_pairs: Vec<(Cow<str>, Cow<str>)> = query_string
-                    .split('&')
-                    .map(|s| {
-                        let mut split = s.splitn(2, '=');
-                        (
-                            Cow::from(split.next().unwrap_or_default()),
-                            Cow::from(split.next().unwrap_or_default()),
-                        )
-                    })
-                    .collect();
-
-                for (key, value) in query_pairs {
-                    pairs.append_pair(&key, &value);
-                }
-            }
```

Comment:
This section got changed above to use actual urlencoded parsing, which prevents a bug in which parameters get double quoted.  If you create an `Options` with a value that includes an `&`, that shows up as `%26` when you serialize `deepgram::common::Options`, but if you pass it to a StreamRequestBuilder`, it serializes as `%2526` (where `%25` is an escaped form of `%` itself).